src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/CommentForm.kt

```diff
@@ -34,7 +34,8 @@ class CommentForm(
     fun enterCommentText(
         comment: String
     ): CommentForm {
-        driver.findElement(By.id("comment")).sendKeys(comment)
+        RichTextEditorTextArea(driver, driver.findElement(By.id("comment")))
```

Comment:
This is a very interesting change from the perspective of the lib clients. Let's document it in the `CHANGELOG.md`.
IMHO we should promote this change to a behavioral contract, ie. we:
```
### Added
- Support RTE with a fallback to plain-text editor. Resolve [JPERF-XXX].
```
This will mean the feature will be available in the next minor release (3.4.0). Thanks to SemVer, it will communicate to clients that they can use 3.4.0 and higher to rely on this feature and it will not be silently removed. They'll be able to remove the RTE config from the setup phase without fear.

Reply:
sure